lms/djangoapps/courseware/date_summary.py

```diff
@@ -107,7 +108,7 @@ def _format_date(self):
         # 'absolute'. For example, 'absolute' might be "Jan 01, 2020",
         # and if today were December 5th, 2020, 'relative' would be "1
         # month".
-        date_format = _("{relative} ago - {absolute}") if date_has_passed else _("in {relative} - {absolute}")
+        date_format = _(u"{relative} ago - {absolute}") if date_has_passed else _(u"in {relative} - {absolute}")
```

Comment:
There actually are translator comments above this line. I agree that it's difficult to translate, but I think this is probably the best we can do if we want to have natural-language time descriptions.